### PR TITLE
Hide search results when click off

### LINKED ISSUE #440: Search results do not go away when you click on other stuff

### PR DESCRIPTION
**Pull request by ritmatter**
Fixes #440

### CODE REVIEW

File: js/app.jsx [hunk 1]
```diff
@@ -28,9 +29,31 @@ var RepsApp = React.createClass({
...
     auth.loggedIn(function(loggedIn) {
       this.setState({ loggedIn: loggedIn });
     }.bind(this));
+
+    // Make sure search results are only shown if actively searching
+    $(document).click(function() {
+      // Start by hiding everything
+      $('.searchDisplayTable').hide();
+
+      console.log($(':focus'));
```
Comment (krazemon) [inline js/app.jsx]: I'm assuming you meant to get rid of this.

File: js/app.jsx [hunk 2]
```diff
@@ -28,9 +29,31 @@ var RepsApp = React.createClass({
...
+      // Check if expert search is being used
+      if ( $('.expertsearch').is(':focus') || $('.expertSearchResults').is(':focus') ) {
+        console.log('showing expert search');
```
Comment (krazemon) [inline js/app.jsx]: This too,

**Comment by krazemon** [comment]
I found two unnecessary console statements. Otherwise, this LGTM. Feel free to merge it once you fix the log statements and remove the merge conflicts.

File: js/app.jsx (hunk 2 above)
Comment (ritmatter) [inline js/app.jsx]: Yup

File: js/app.jsx (hunk 1 above)
Comment (ritmatter) [inline js/app.jsx]: Yup

**Pull request merged**